agent/nrpe.rb

```diff
@@ -50,8 +51,8 @@ class Nrpe<RPC::Agent
       # Example :
       #          plugin.nrpe.conf_dir = /etc/nagios/nrpe
       #          plugin.nrpe.conf_file = checks.nrpe
-      def self.run(command)
-        nrpe_command = Nrpe.plugin_for_command(command)
+      def self.run(command, *args)
```

Comment:
Aham, then I'll need to change also all tests to make them expect to be executed with command and `[]`

If you agree, I'm happy to change this.